applications/virtual-fly-brain/client/src/shared/subHeader/SearchResult/index.js

```diff
@@ -0,0 +1,117 @@
+import { Box, Typography, Chip, Button } from "@mui/material";
+import React from "react";
+import vars from "../../../theme/variables";
+import { Search } from "../../../icons";
+import ArrowOutwardIcon from '@mui/icons-material/ArrowOutward';
+
+const { secondaryBg, searchBoxBg, whiteColor, searchHeadingColor, listHover } = vars;
+
+export const SearchResult = ({ getOptionProps, groupedOptions, chipColors, handleResultSelection }) => {
+  return (
+    <Box sx={{
+      py: '1rem',
+      px: '0.75rem',
+    }}>
+      <Typography variant="body2" sx={{
+        fontSize: '0.75rem',
+        lineHeight: '133%',
+        letterSpacing: '-0.005em',
+        fontWeight: 500,
+        color: searchHeadingColor
+      }}>
+        Suggested results
+      </Typography>
+      <Box mt={1.5}>
+        {groupedOptions?.map((option, index) => (
+          <Box key={`groupedOptions-${index}`} {...getOptionProps({ option, index })}>
+            <Box onClick={() => handleResultSelection(option)} sx={{
+              position: 'relative',
+              p: 0.5,
+              borderRadius: 1,
+              display: 'flex',
+              alignItems: 'center',
+
+              '& > button': {
+                opacity: 1,
+                transition: 'all ease-in-out .3s'
+              },
+
+              '&:not(:hover)': {
+                '& > button': {
+                  opacity: 0
+                }
+              },
+
+              '&:hover': {
+                backgroundColor: secondaryBg,
+                cursor: 'pointer',
+              }
+            }}>
+              <Box sx={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+                width: '1.5rem',
+                height: '1.5rem',
+                borderRadius: 1,
+                background: searchBoxBg,
+              }}>
+                <Search size={12} />
+              </Box>
+
+              <Typography variant="body2" sx={{
+                fontSize: '0.75rem',
+                whiteSpace: 'nowrap',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                color: searchHeadingColor,
+                px: 1
+              }}>
+                {option?.label}
+              </Typography>
+
+              <Box sx={{
+                ml: 'auto',
+                display: 'flex',
+                alignItems: 'center',
+                width: '30%',
```

Comment:
@enicolasgomez please remove the width from here

Reply:
@vidhya-metacell If we don't control the width of this element, the Tags overflow the full row displaying results. 

The Search results are coming back with a lot of Tags, more than can be displayed at once.
![image](https://github.com/MetaCell/virtual-fly-brain/assets/4562825/9e924595-55a9-4292-9afe-2aca01f726e5)